Deepgram/Clients/Manage/v1/Client.cs

```diff
@@ -0,0 +1,253 @@
+        await PatchAsync<MessageResponse>(
+            $"{UriSegments.PROJECTS}/{projectId}",
+            RequestContentUtil.CreatePayload(updateProjectSchema), cancellationToken);
+
+    /// <summary>
+    /// Deletes a project, no response will be returned
+    /// </summary>
+    /// <param name="projectId">Id of project</param>
+    // No response expected
+    public async Task DeleteProject(string projectId, CancellationToken cancellationToken = default) =>
+     await DeleteAsync($"{UriSegments.PROJECTS}/{projectId}", cancellationToken);
+
+    /// <summary>
+    /// leave project associated with the project Id
+    /// </summary>
+    /// <param name="projectId">Id of project</param>
+    /// <returns><see cref="MessageResponse"/></returns>
+    public async Task<MessageResponse> LeaveProject(string projectId, CancellationToken cancellationToken = default) =>
+        await DeleteAsync<MessageResponse>($"{UriSegments.PROJECTS}/{projectId}/leave", cancellationToken);
+
+    #endregion
+
+    #region ProjectKeys
+
+    /// <summary>
+    /// Get the keys associated with the project
+    /// </summary>
+    /// <param name="projectId">Id of project</param>
+    /// <returns><see cref="KeysResponse"/></returns>
+    public async Task<KeysResponse> GetKeys(string projectId, CancellationToken cancellationToken = default) =>
+        await GetAsync<KeysResponse>($"{UriSegments.PROJECTS}/{projectId}/{UriSegments.KEYS}", cancellationToken);
+
+    /// <summary>
+    /// Get details of key associated with the key ID
+    /// </summary>
+    /// <param name="projectId">Id of project</param>
+    /// <param name="keyId">Id of key</param>
+    /// <returns><see cref="KeyScopeResponse"/></returns>
+    public async Task<KeyScopeResponse> GetKey(string projectId, string keyId, CancellationToken cancellationToken = default) =>
+        await GetAsync<KeyScopeResponse>($"{UriSegments.PROJECTS}/{projectId}/{UriSegments.KEYS}/{keyId}", cancellationToken);
+
+    /// <summary>
+    /// Create a key in the associated project
+    /// </summary>
+    /// <param name="projectId">Id of project</param>
+    /// <param name="createProjectKeySchema"><see cref="KeySchema"/> for the key to be created</param>
+    /// <returns><see cref="KeyResponse"/></returns>
+    public async Task<KeyResponse> CreateKey(string projectId, KeySchema createProjectKeySchema, CancellationToken cancellationToken = default)
+    {
+        if (createProjectKeySchema.ExpirationDate is not null && createProjectKeySchema.TimeToLiveInSeconds is not null)
+        {
+            Log.CreateKeyError(_logger, createProjectKeySchema);
+            throw new ArgumentException("Both ExpirationDate and TimeToLiveInSeconds is set. set either one but not both");
+        }
+
+        return await PostAsync<KeyResponse>(
+                     $"{UriSegments.PROJECTS}/{projectId}/keys",
+                     RequestContentUtil.CreatePayload(createProjectKeySchema), cancellationToken);

```

Comment:
Please see this issue as they are similar:
https://github.com/deepgram/deepgram-dotnet-sdk/pull/237#discussion_r1496080055

Will address the TODOs for Logging in these issues since I haven't thought about how to address it yet: https://github.com/deepgram/deepgram-dotnet-sdk/issues/210 and https://github.com/deepgram/deepgram-dotnet-sdk/issues/219